### DISCUSSION TITLE
Check For Played History

### DISCUSSION THREAD

**Discussion by ghost** [sct/overseerr#2298] [Feature Requests]
Greetings!

Would it be possible to have overseerr check to see if a requested media item has been played since plex now supports play history? I think it would be a great addition to verify requests are getting watched. Currently this can be done manually with checking play history on the individual item or via Tautulli but a check built into overseerr would be nice.

Thanks!

**Comment by TheCatLady** [comment]
#2230 adds Tautulli integration.  It doesn't display play history, but does show whether or not an item has been played, and links to the play history in Tautulli.

**Comment by ghost** [comment]
> 

Awesome! That's exactly what I was looking for, thanks!
